### ISSUE TITLE
Installing nbextensions

### ISSUE THREAD

**Issue by ifuchs** [holzschu/Carnets#70]
What are the steps to install and enable an nbextension (e.g., code prettify)? I tried !jupyter nbextension but that gave command nbextension not found.

**Comment by holzschu** [comment]
nbextensions are all installed by default. To enable individual extensions, use the menu Edit -> "nbextensions config".  

Alternatively, if you opened a directory (using the usual method: click on "Select", click next to the directory, click "Open"), you have a tab at the top for "nbextensions". It will open the same page.

**Comment by ifuchs** [comment]
Thanks! I enabled the code_prettify extension but when I tried to use it I got code_prettify module not found error.

**Comment by holzschu** [comment]
In the documentation of code_prettify, at the bottom, it says you have to install yapf first, with "!pip install yapf" (it is in the section Prerequisites).

**Comment by ifuchs** [comment]
Ha. I had just done that install except it was on Pyto. It works fine now.  I am doing a feature comparison of Pythonista, Pyto, and Carnets. If only there were a way to merge the best of each into one app :-)

**Comment by holzschu** [comment]
I would say that each app has its own advantages and benefits. Pythonista has a tight integration with iOS, allowing access to the camera and user interface. Carnets is focused on Jupyter notebooks, with the code / markdown / html being side by side with the results. And Pyto brings the latest Python version and a different user interface. You could say they target three different publics.

**Comment by holzschu** [comment]
To go back to your question on accessing the microphone and camera, that is something that is both easy to do and makes sense in Pythonista, and is both hard to do and makes less sense with Jupyter notebooks. On the other hand, there are other things you can do with notebooks, like combine text, equations and curves in a single document.

**Comment by ifuchs** [comment]
I agree completely. The NB interface is best for certain use cases and the Pythonista and Pyto UI is better for others. The IOS integration is useful to those writing IOS apps but not so much for pure python coders. Looking at the major differences between these apps, the main thing that is of interest to the largest cross-section of users is the availability of Pandas and SciPy (including Scikit modules). As you know, Pythonista has none of these, Carnets has Pandas but no SciPy and Pyto has both.

**Comment by holzschu** [comment]
Carnets will have ScipPy eventually, but probably as a paid extension. The main difference between Carnets and the other two is that you can close or terminate a notebook, then restart it without restarting the app. In terms of user functionality that's not much, but from the app programmer view point that's a lot of work.

**Comment by ifuchs** [comment]
Would that mean the end of open source or just the paid AppStore version?

**Comment by holzschu** [comment]
That would still be open source, and everything that does not require SciPy would still be in the free version, but everything that requires SciPy would be in the paid version. I'm still working on the details.
